2-Draft-Accepted/RFCNNNN-Support-Experimental-Features.md

```diff
@@ -80,6 +83,7 @@ The `JSON` schema should look like this:
   ]
 }
 ```
+> NOTE: Should we consider reading experimental feature lists also from the per-user configuration file?
 
 PowerShell reads the experimental feature list from the configuration file only once when starting up.
```

Comment:
I think we should be pretty explicit here. The implication is that since we read the config file once during startup, it's implied that changing experimental features at runtime, I think that should be an explicit statement. Also, we should be explicit that the config file is read before engine start

Reply:
Updated the words to be more explicit.